docs/zip_html.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
this script doesn't seem to be used... and would probably be better off as a call to `zip` or `tar` anyway.